modules/rosa-policy-security-regulation-compliance.adoc

```diff
@@ -71,7 +71,8 @@ Any issues that may be discovered are prioritized based on severity. Any issues
 
 | SOC 3 | Yes | Yes
 
-| FedRAMP High^[1]^ | Yes (link:https://aws.amazon.com/govcloud-us/?whats-new-ess.sort-by=item.additionalFields.postDateTime&whats-new-ess.sort-order=desc[GovCloud] requisite) | No
+| FedRAMP High^[2]^ | Yes (link:https://aws.amazon.com/govcloud-us/?whats-new-ess.sort-by=item.additionalFields.postDateTime&whats-new-ess.sort-order=desc[GovCloud] requisite) | No
 
 |===
-1. For more information about ROSA on GovCloud, see the link:https://marketplace.fedramp.gov/products/FR2102031769[FedRAMP Marketplace ROSA Agency] and link:https://marketplace.fedramp.gov/products/FR2102031769A[ROSA JAB listings].
\ No newline at end of file
+1. For more information about Red Hat's HIPAA Qualified ROSA offerings, see the link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#hipaa-overview-13[Red Hat Compliance Activities and Government Standards page].
```

Comment:
```suggestion
1. For more information about Red Hat's HIPAA Qualified ROSA offerings, see  link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#hipaa-overview-13[HIPAA Overview].
```

Reply:
Per discussion, kept the "the" but changed the link name as requested.